andela/theoden-cfh
# Bug/fix gulpfile

#### What does this PR do?
Fix issues with app not rendering on heroku
#### Description of Task to be completed?
- Add sequence of tasks in gulpfile that moves bootstrap, angular-route and angular-util to required locations.
- Add gulp-clean to package dependencies.
#### How should this be manually tested?
Delete `public/lib` folder on your local repo and start app on the terminal with `gulp`. On the browser, it should render a blank screen, with 404 errors on the browser console. Next, run `gulp install` on the terminal. Afterwards, start app again. This time, it should render on the browser with no 404 errors.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#### Questions:
